skills/gh-address-comments/scripts/thread_context.py

```
@@ -0,0 +1,199 @@
+        if args.pr:
+            sha = get_head_sha(owner, repo, args.pr)
+        else:
+            pr_num = data.get("pull_request", {}).get("number")
+            if not pr_num:
+                print("Cannot determine PR number — pass --pr NUMBER", file=sys.stderr)
+                sys.exit(1)
+            sha = get_head_sha(owner, repo, pr_num)
+
+        file_lines = fetch_file_at_ref(owner, repo, path, sha)
+        show_context(thread, file_lines, args.lines)
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)